pilot/pkg/features/pilot.go

```diff
@@ -524,6 +524,9 @@ var (
 		true,
 		"If false, TCP probes will not be rewritten and therefor always succeed when a sidecar is used.",
 	).Get()
+
+	EnableInternalTrafficPolicy = env.RegisterBoolVar("PILOT_USE_KUBERNETES_INTERNAL_TRAFFIC_POLICY", false,
```

Comment:
nit: Can we remove `PILOT` from the variable? Maybe something like `ENABLE_KUBERNETES_INTERNAL_TRAFFIC_POLICY`?

Reply:
Thanks for your advice,  i think calling it **PILOT_ENABLE_KUBERNETES_INTERNAL_TRAFFIC_POL**  might be more clear and 
same as most existing env naming formats.